### ISSUE TITLE
Document Issue: File path is incorrect in " Upgrading from AngularJS "

### ISSUE THREAD

**Issue by JimmyLovely** [angular/angular#25904]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior

There is a document issue. 

```
Document ->  Techniques  ->  Upgrading from AngularJS  -> Upgrading Instructions -> PhoneCat Upgrade Tutorial -> Bootstrapping a hybrid PhoneCat
```

Link: https://angular.io/guide/upgrade#bootstrapping-a-hybrid-phonecat

"Finally, bootstrap the AppModule in <b>src/main.ts</b>. This file has been configured as the application entrypoint in systemjs.config.js, so it is already being loaded by the browser."

## Expected behavior

"Finally, bootstrap the AppModule in <b>app/main.ts</b>. This file has been configured as the application entrypoint in systemjs.config.js, so it is already being loaded by the browser."

## Minimal reproduction of the problem with instructions

[Attachment](https://1drv.ms/u/s!AtzEKIo6Vmm8kZQfFJO4H09_Rp6BBQ)

## What is the motivation / use case for changing the behavior?

it  <b>confuses</b> reader.

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- [x] Chrome (desktop) version XX
- [] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by jenniferfell** [comment]
Thanks for the screen shot and clear details. 

Notes for myself: 
"Finally, bootstrap the AppModule in src/main.ts. This file has been configured as the application entrypoint in systemjs.config.js, so it is already being loaded by the browser."

Code example below has title: app/main.ts

**Comment by georgewilde** [comment]
Hi Jennifer,
Is it correct that I've just picked this up and put a PR in for it? As you had commented I wasn't sure if it was something you were already fixing?

**Comment by dariobraun** [comment]
Looks like this issue can be closed.

**Comment by gkalpak** [comment]
Thx, @dariobraun 👍
This was fixed in #26072.
